src/chmfile.cpp

```diff
@@ -521,11 +523,14 @@ bool CHMFile::BinaryIndex(CHMListCtrl& toBuild, const wxCSConv& cv)
 
                     offset += sizeof(uint32_t);
                     spaceLeft -= sizeof(uint32_t);
-                }
+                }                
             }
 
             if (bt_ui.length < offset + 8)
+            {
```

Comment:
This breaks the existing coding style that does `if (condition) {`. Again, please use `clang-format -i` on the source code before submitting the code changes.